src/Service/Validation/RegisterTokenValidator.cs

```diff
@@ -30,4 +31,18 @@ public static void ValidateAttestation(RegisterToken token, IFeaturesContext fea
             throw new ApiException("attestation_not_supported_on_plan", "Attestation type not supported on your plan", 400);
         }
     }
+
+    public static void Validate(this Token token, TimeProvider timeProvider)
```

Comment:
If we don't want this method to be a part of `FidoService` and receive `TimeProvider` as a contextual parameter, then this static method can just simply take a `DateTimeOffset now` parameter instead.